xml/tuning_cgroups.xml

```diff
@@ -66,6 +66,12 @@
    slice, scope, socket, mount, or swap name. Replace properties with
    one or more of the following:
   </para>
+  <para>
```

Comment:
This is a bit awkward. I suggest moving the paragraph "Limitations to <literal>cgroups</literal>..." after "Beware that resource consumption...", and make "Beware that resource consumption..." a Note, like this:


&lt;note>
&lt;title>Resource Consumption Depends on Workload&lt;/title>
&lt;para>
...
&lt;/para>
&lt;/note>

Reply:
I am happy to make the backports, if you like.

Reply:
Thanks for the review. I've force-pushed your proposals. Feel free to edit it as necessary for the common document formatting, I don't care that much as long as it conveys the message.